cpp/examples/ode_secir_ageres.cpp

```diff
@@ -97,25 +98,6 @@ int main()
     if (print_to_terminal) {
 
         std::vector<std::string> vars = {"S", "E", "C", "C_confirmed", "I", "I_confirmed", "H", "U", "R", "D"};
-        printf("\n # t");
```

Comment:
why remove everything here? I think we can leave this here until we have a new feature for plotting

Reply:
Because the first part was mostly for debugging the timesteps. There are more prints afterwards that show the results of the last timestep for each compartment and agegroup